### PR TITLE
Blog Refactor

### PR DESCRIPTION
**Pull request by ghislaineguerin**
This blog cleanup will focus mostly on removing unnecessary styles and maintaining elements consistency across the site so that changes can be applied globally.

It will also include a new component:
### Blog footer
image

### CODE REVIEW

File: blog/_posts/2018-07-24-from-spa-to-pwa.md [hunk 1]
```diff
@@ -554,7 +554,7 @@ to execute -
 completes in
 [around 1min](https://travis-ci.org/simplabs/breethe-client/jobs/403597086#L683).
 
-#### Outlook
+### Outlook
 
 This post and the
 [previous one](/blog/ember/2018/07/03/building-a-pwa-with-glimmer-js.html) have given
```
Comment (marcoow) [inline blog/_posts/2018-07-24-from-spa-to-pwa.md]: Should this be changed for **all** posts?

Comment (ghislaineguerin) [reply blog/_posts/2018-07-24-from-spa-to-pwa.md]: @marcoow yes, you are right, I actually want to leave this for after we release the website, as many of the posts we have are lacking h2 tags and our headings are not SEO optimized. I'll just discard this change.

**Comment by ghislaineguerin** [comment]
@marcoow this is once again ready for review.

**Review by marcoow — APPROVED** [review]

**Comment by marcoow** [comment]
I don't quite get what the problem with the build is preventing it from even starting - merging anyway 🚀

**Pull request merged**